js/goproblems.js

```diff
@@ -102,6 +102,32 @@ go.problems.Player.prototype = {
       this.callback = callback;
     }
 
+    let autosize = false;
+
+    if (configuration.stoneSize === 'auto') {
+      autosize = true;
+
+      let winFixer = 0;
+
+      if (window.outerWidth <= 600) {
+        let lastStep = 6;
+
+        for (let i = 600 - window.outerWidth; i > 0; i -= 100) {
+          lastStep += 1.2;
+        }
+
+        winFixer = window.outerWidth / 100 * lastStep;
+      }
+
+      console.log(winFixer);
```

Comment:
there's probably debug flag or something like this in configuration
so such messages should be exposed in case of debug flag was set